### DISCUSSION TITLE
Welcome to SubmersedVR Discussions!

### DISCUSSION THREAD

**Discussion by Okabintaro** [Okabintaro/SubmersedVR#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NeonPixels** [comment]
Well, hello there!
I'm new to VR, and this mod has put Subnautica (already one of my favourite games) in my Top Tier of VR games.
I am a software engineer, and do dabble in modding myself from time to time, so I'm glad to contribute in any way I can.

**Reply by Readied** [reply]
<sub>let me be your student</sub>
